src/d3.chart.js

```diff
@@ -84,15 +84,26 @@ class d3chart {
     * Set up chart dimensions
     */
     getDimensions(){
-        const width = parseInt(this.selection.node().offsetWidth) - this.cfg.margin.left - this.cfg.margin.right;
-        const height = parseInt(this.selection.node().offsetHeight) - this.cfg.margin.top - this.cfg.margin.bottom;
-
-        if(this.cfg.fixedSize && width === 0) {
-            console.info("Fixed size set - getDimensions skipped")
-        }else{
+        function isHidden(node) {
```

Comment:
this will be changed to allow passing visibility detection function from parent component

Reply:
ok